Script.php

```diff
@@ -682,50 +663,48 @@ function checksize($file)
  
  
  
- 
- 
- // ############################### Submit Assignment ##################################
-    if (!empty($_POST["frm_submitlab"])) {
+// ############################### Submit Assignment ##################################
+if (!empty($_POST["frm_submitlab"])) {
         
      
         
-     $lab_id=mysqli_real_escape_string($con,$_POST["lab_id"]);
-     $student_id=$_POST["student_id"];
-        $group_id=$_POST["group_id"];
-     
+  $lab_id=mysqli_real_escape_string($con,$_POST["lab_id"]);
+  $student_id=$_POST["student_id"];
+     $group_id=$_POST["group_id"];
+  
 
-       $instructions=mysqli_real_escape_string($con,$_POST["instructions"]);
-       $title=mysqli_real_escape_string($con,$_POST["title"]);
-       
-         $url=mysqli_real_escape_string($con,$_POST["url"]);
-       
-       
-       $deadline=$deadlinedate." ".$deadlinetime;
-      $date=  date("Y-m-d H:i");
-            
-       
-       
- // GET UPLOADED FILES
-       
-     
+    $instructions=mysqli_real_escape_string($con,$_POST["instructions"]);
+    $title=mysqli_real_escape_string($con,$_POST["title"]);
+    
+      $url=mysqli_real_escape_string($con,$_POST["url"]);
+    
+    
+    $deadline=$deadlinedate." ".$deadlinetime;
```

Comment:
@Teecloudy 

Are these two lines properly indented?

**Hui**

Reply:
These were part of original lines. The form from the instructor's side contains input '$deadlinedat' and '$deadlinetime' separately and the writer combined both values into one ($deadline) before inserting into the database. However this was a challenge when trying to retrieve it so the instructor can edit the report, so we split it again using 
$_SESSION['Date'] = strstr($Deadline, ' ', true);
$_SESSION['Time'] = strstr($Deadline, ' '); 
at line 120 and 121 in Courses.php